internal/cli/atlas_clusters_create.go

```diff
@@ -85,17 +85,29 @@ func (opts *atlasClustersCreateOpts) newCluster() (*atlas.Cluster, error) {
 		cluster.Name = opts.name
 	}
 
-	cluster.Labels = []atlas.Label{
-		{
-			Key:   config.LabelKey,
-			Value: config.LabelValue,
-		},
-	}
+	opts.updateLabels(cluster)
 
 	cluster.GroupID = opts.ProjectID()
 	return cluster, nil
 }
 
+func (opts *atlasClustersCreateOpts) updateLabels(out *atlas.Cluster) {
```

Comment:
[nit] this method doesn't use `opts` in its body, so you could make it a generic function like 
```suggestion
func  updateLabels(out *atlas.Cluster) {
```
This also let's you reuse the same function in both create an update